rpm/nemo-qml-plugin-systemsettings.spec

```diff
@@ -64,7 +64,7 @@ Summary: Translation source for %{name}
 %setup -q -n %{name}-%{version}
 
 %build
-%qmake5 "VERSION=%{version}"
+%qmake5 "VERSION=%{version}" DEVMODE=1
```

Comment:
Could you make this opt-out instead of opt-in. Besides this =1 part is quite random way to do it.